docs/openapi.yaml

```diff
@@ -79,10 +79,15 @@ definitions:
         minItems: 1
         items:
           type: string
+      type:
+        description: |-
+          Request type metadata
+        type: string
     required:
       - user
       - products
-  PredictionResponse:
+      - type
```

Comment:
do we need to know the type?

we have a specific endpoint and payload for this type of request, seems unnecessary.